Sources/FTPropertyWrappers/Serialized.swift

```diff
@@ -14,15 +14,34 @@ public final class Serialized<Value> {
         }
     }
 
-    /// Did set observer for stored property. Notice, that didSet event is called on the synchronization queue. You should free this thread asap with async call, since complex operations would slow down sync access to the property.
+
+    /// Did set observer for stored property. Notice, that didSet event is called on the synchronization queue.
+    /// You should free this thread asap with async call, since complex operations would slow down sync access
```

Comment:
```suggestion
    /// You should free this thread as soon as possible with async call, since complex operations would slow down the synchrounous access
```